src/gantry/k8s.py

```
@@ -1152,3 +1158,120 @@ def get_pod_logs(
+        if field == "age":
+            now = time.time()
+            if value and value[0] in ("<", ">"):
+                op = value[0]
+                secs = _parse_duration(value[1:])
+                if secs is not None:
+                    cutoff = now - secs
+                    if op == "<":
+                        return lambda r, c=cutoff: (r.get("age_seconds") or 0) > c
+                    return lambda r, c=cutoff: 0 < (r.get("age_seconds") or 0) < c
+            # Malformed age expression — don't match anything.
+            return lambda r: False
```

Ruff (0.15.12)

[warning] Unused lambda argument: `r`

(ARG005)